examples/cisco/demo_bgp.pp

```diff
@@ -73,7 +73,11 @@
     enforce_first_as                       => false,
     event_history_cli                      => 'size_small',
     event_history_detail                   => 'size_medium',
+    # event_history_errors is not supported on all platforms
+    # event_history_errors                   => 'size_small',
```

Comment:
Good point.  There isn't one but I think it would be good to add one under https://github.com/cisco/cisco-network-puppet-module/tree/develop/lib/puppet/parser/functions.

Just add a new function say `image_version_get.rb`.

You can use `platform_get.rb` as an example.  When you add the new function, you will need to restart your puppet master to get it to detect the new function.